engine/helpers.go

```diff
@@ -737,25 +737,35 @@ func (bot *Engine) GetAllEnabledExchangeAccountInfo() AllEnabledExchangeAccounts
 	var response AllEnabledExchangeAccounts
 	exchanges := bot.GetExchanges()
 	for x := range exchanges {
-		if exchanges[x] != nil && exchanges[x].IsEnabled() {
-			if !exchanges[x].GetAuthenticatedAPISupport(exchange.RestAuthentication) {
-				if bot.Settings.Verbose {
-					log.Debugf(log.ExchangeSys,
-						"GetAllEnabledExchangeAccountInfo: Skipping %s due to disabled authenticated API support.\n",
-						exchanges[x].GetName())
-				}
-				continue
+		if exchanges[x] == nil || !exchanges[x].IsEnabled() {
+			continue
+		}
+		if !exchanges[x].GetAuthenticatedAPISupport(exchange.RestAuthentication) {
+			if bot.Settings.Verbose {
+				log.Debugf(log.ExchangeSys,
+					"GetAllEnabledExchangeAccountInfo: Skipping %s due to disabled authenticated API support.\n",
+					exchanges[x].GetName())
 			}
-			accountInfo, err := exchanges[x].FetchAccountInfo()
+			continue
+		}
+		assetTypes := exchanges[x].GetAssetTypes()
+		var exchangeHoldings account.Holdings
+		for y := range assetTypes {
```

Comment:
Tested using Binance: 

[DEBUG] | 10/02/2021 17:02:26 | Portfolio: Adding new exchange address: Binance, BTC, xxxx.xxxx, Exchange (with split balances in both spot and futures). Nice!

Reply:
xxxx.xxxx 4 digit btc balance :D